website/notifications/emails.py

```diff
@@ -78,7 +80,17 @@ def email_digest(subscribed_user_ids, uid, event, **context):
 }
 
 
-def notify(uid, event, **context):
+def notify(uid, event, user, node, timestamp, **context):
+    """
+    :param uid: node's id
+    :param event: type of notification
+    :param user: user triggering notification
+    :param node: the node
+    :param timestamp: time
+    :param context: optional variables specific to templates
+        target_user: used with comment_replies
```

Comment:
I was explicitly passing `comment_replies` as the event type before, but then it would only notify the `target_user` and not other users who were interested in comments on that project. So the solution I came up with was to always pass `comments` as the event type and use the `comment_replies` template for the user whose comment was replied to.